packages/cli/index.js

```diff
@@ -311,13 +315,33 @@ program
     const defaultOutputRoot = path.join(rootFolder, '_site');
     const outputFolder = output ? path.resolve(process.cwd(), output) : defaultOutputRoot;
     new Site(rootFolder, outputFolder, undefined, undefined, options.siteConfig)
-      .generate(baseUrl)
+      .generate(baseUrl, options.versions)
       .then(() => {
         logger.info('Build success!');
       })
       .catch(handleError);
   });
 
+program
+  .command('archive [versionName] [archivePath]')
+  .alias('ar')
+  .option('-s, --site-config <file>', 'specify the site config file (default: site.json)')
+  .description('archive a version of the site, which is not affected by later changes to the site')
+  .action((versionName, userSpecifiedArchivePath, options) => {
+    if (!versionName) {
+      logger.error('Please specify a name for the archived version.');
```

Comment:
Maybe can throw an error here so that it will stop the archiving process if the version name is not specified?

Currently, it will be archived as `undefined`:
![image](https://user-images.githubusercontent.com/47470981/164441685-3e90f60b-f6af-47c9-817c-b037907ac12b.png)